flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/sink/TestIcebergFilesCommitter.java

```diff
@@ -242,21 +246,21 @@ public void testOrderedEventsBetweenCheckpoints() throws Exception {
       RowData row1 = SimpleDataUtil.createRowData(1, "hello");
       DataFile dataFile1 = writeDataFile("data-1", ImmutableList.of(row1));
 
-      harness.processElement(of(dataFile1), ++timestamp);
+      long firstCheckpointId = 1;
+      harness.processElement(of(firstCheckpointId, dataFile1), ++timestamp);
```

Comment:
@pvary When apply V1.20 patch, a misalignment occurred. 

But I don't know what caused it, so I fixed it manually.

Reply:
I think this is related to @rodmeneses's last commit.
Could you please check @rodmeneses?
Thanks,
Peter

Reply:
https://github.com/apache/iceberg/commit/e6f8ab9950267c18734ba0cb2ce29867790e24fe#diff-4f79d5342dad9de47d561297e1af60aac21a498f22144ad378dc762affdbfa27L142-L143

Yes, thanks for your reminder.
It should be the modification of `TestIcebergFilesCommitter.java` that caused the inconsistent line numbers.